dotnet-projects/MultipleFrameworkTarget/MultipleFrameworkTargetConsole.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.6.33829.357
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MultipleFrameworkTargetConsole", "..\..\..\..\..\MultipleFrameworkTargetConsole\MultipleFrameworkTargetConsole.csproj", "{1291FFE2-345C-4576-B242-D670EC9C8DA3}"
```

Comment:
Something is messed up in this solution file and that is causing the build to fail. If you look at the path that the solution has for the MultipleFrameworkTargetConsole.csproj, it is looking for it five directories up from the solution location

Reply:
@DharaVaghasiya please have a look at this what @jshergal pointed out. This looks definitely like the problem :-)